lib/pages/art/MasterArtPage.dart

```diff
@@ -46,18 +46,24 @@ class _MasterArtPageState extends State<MasterArtPage> {
         setState(() {
           listInstallation.add(
             Installation(
-                result.data["installations"][i]["title"],
-                result.data["installations"][i]["desc"],
-                zone: result.data["installations"][i]["zone"],
-                imgUrl: result.data["installations"][i]["image"]["url"],
-                location: {'latitude': result.data["installations"][i]["location"]["latitude"],
-                'longitude': result.data["installations"][i]["location"]["longitude"],
-                },
-                locationRoom: result.data["installations"][i]["locationroom"],
-                profiles: []
+              result.data["installations"][i]["title"],
+              result.data["installations"][i]["desc"],
+              zone: result.data["installations"][i]["zone"],
+              imgUrl: result.data["installations"][i]["image"] == null
+                  ? 'https://via.placeholder.com/350'
+                  : result.data["installations"][i]["image"]["url"],
+              location: {
+                'latitude': result.data["installations"][i]["location"] == null
+                    ? 0.0
+                    : result.data["installations"][i]["location"]["latitude"],
+                'longitude': result.data["installations"][i]["location"] == null
+                    ? 0.0
+                    : result.data["installations"][i]["location"]["longitude"],
+              },
+              locationRoom: result.data["installations"][i]["locationroom"],
+              profiles: [],
```

Comment:
you need to grab the profile of the artists here to pass into the art detail page. To do this, check if result.data["installations"][i]["profiles"] exists and loop to create a list of profiles to add to the installations.

You dont have to fix this for this review if someone on your team is working on this

Reply:
will fix this with team later.